### DISCUSSION TITLE
Welcome to gamesense.pub Discussions!

### DISCUSSION THREAD

**Discussion by h4xrOx** [h4xrOx/gamesense.bar#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 1580155hsg** [comment]
**Please** _invite_ me to _**GAMESENSE**_  .
**Please** _listen_ to me :"**_skeet skeet_** **_number 1_** **_cheat_**"
**_I LOVE YOU WISH_**
